gitops/AzureDevOps/README.md

```diff
@@ -0,0 +1,202 @@
+# Approach
+
+Taking into account that devops-as-code provides convenient wrapper for *NIX and Windows systems, it is very easy to using together with different CI tools.
+
+It is very easy to integrate devops-as-code into your git flow in Azure DevOps you should do the following:
+
+1) Add wrappers to your project. You can either copy/paste them from another project, or execute `xl wrapper` command in the root of your project to generate them.
+2) Define devops-as-code YAML in your project's directory.  
+3) Define different pipelines depending on the OS. On Windows you should use a `script` step or a `bash` step in your `azure-pipeline.yml` file and in there you
+can use your different devops-as-code YAML file(s)
+    * Windows:
+        ```yaml
+          ....
+          - bash: |
+             # Write your commands here
+             
+             # Use the environment variables input below to pass secret variables to this script
+             
+             sleep 120
```

Comment:
I have no idea why we put sleep here at all. Especially, how is it related to this example? :)